LocalSettings.php

```diff
@@ -4411,6 +4411,99 @@
 			NS_MAIN => true,
 			NS_USER => true,
 		),
+		'wmgVisualEditorAvailableNamespaces' => array(
```

Comment:
This array isn’t closed

Reply:
@MacFan4000 looks fine to me.. I’m confused

Reply:
It’s an array of the setting within an array of the setting.

Reply:
@JohnFLewis I am still very confused. I’ve given you temporary access to my repo fork - please fix it for me. Sorry, but I just don’t get what I screwed up.